packages/shared-aco/src/flp/Permissions.ts

```diff
@@ -3,7 +3,7 @@ import type { FolderLevelPermission, FolderPermission } from "~/flp/flp.types";
 export class Permissions {
     public static create(
         permissions?: FolderPermission[],
-        parentFlp?: FolderLevelPermission | null
```

Comment:
Didn't we do `FolderLevelPermission | null` in our of our last reviews, just to replace what's on the right here?

Reply:
Yes, you are correct. I had to revert and select certain fields because the `FolderLevelPermission` entity is utilised solely by the backend. In the frontend, we are using `Folder`. 

See: `packages/app-aco/src/features/folders/updateFolder/UpdateFolderRepositoryWithPermissionsChange.ts`